libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/TripService.kt

```diff
@@ -1,5 +1,10 @@
 package com.mapbox.navigation.base.trip
 
+import android.location.Location
+const val NAVIGATION_NOTIFICATION_ID = 5678
+
+data class NavigationTripDescriptor(val notificationId: Int = NAVIGATION_NOTIFICATION_ID, val routeProgress: RouteProgress, val rawLocation: Location, val enhansedLocation: Location)
```

Comment:
Should we create a new package `model` under `trip` and place this data class there?

Reply:
NIT Typo * enhansedLocation should be `enhancedLocation`

Reply:
add a Constants.kt file that contains all the constants